astacus/coordinator/plugins/zookeeper.py

```diff
@@ -124,6 +126,11 @@ def transaction(self) -> ZooKeeperTransaction:
         raise NotImplementedError
 
 
+class ZooKeeperUser(AstacusModel):
+    username: str
+    password: SecretStr
+
```

Comment:
I'd rather not have pydantic models go through all code, and restrict it to IO (config & ipc) → Having a model for config and a separate dataclass for the internals would be cleaner and avoid some future headache.

Reply:
Makes sense, since we have to standard naming for the pydantic -> dataclass translation chose `to_dataclass`.